### PR TITLE
Merge back release branch

### PR DESCRIPTION
**Pull request by brizental**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Unreleased changes
 
-[Full changelog](https://github.com/mozilla/glean.js/compare/v0.1.1...main)
+[Full changelog](https://github.com/mozilla/glean.js/compare/v0.2.0...main)
+# v0.2.0 (2021-02-23)
```
Comment (Dexterp37) [inline CHANGELOG.md:4]:
```suggestion
[Full changelog](https://github.com/mozilla/glean.js/compare/v0.2.0...main)

# v0.2.0 (2021-02-23)
```

**Review by Dexterp37 — APPROVED** [review]
r+wc

File: CHANGELOG.md (hunk 1 above)
Comment (brizental) [reply CHANGELOG.md:4]: Hum, should probably change this in the `publish-release.sh` script... This is all generated after all. 1sec.

Comment (brizental) [reply CHANGELOG.md:4]: https://github.com/mozilla/glean.js/pull/88

**Pull request merged**